app/helpers/toolbar_helper.rb

```diff
@@ -191,19 +191,20 @@ def toolbar_button_normal(props, view_button = false)
   # attributes in html
   #
   def data_hash_keys(props)
-    hsh = %i(pressed popup console_url prompt explorer onwhen url_parms url).each_with_object({}) do |key, h|
-      h["data-#{key}"] = props[key] if props.key?(key)
+    %i(pressed popup console_url prompt explorer onwhen url_parms url).each_with_object({}) do |key, h|
+      h[key] = props[key] if props.key?(key)
     end
-    hsh
   end
 
   # Calculate common html tag keys and values from toolbar button definition
   #
   def prepare_tag_keys(props)
-    h = data_hash_keys(props)
+    h = {
+      'data'       => data_hash_keys(props),
```

Comment:
@martinpovolny as long as it stays mainly static, I don't really see a problem - we're only ever explicitly setting `enabled`, `disabled`, `toogle`, `click`, `confirm`, none of which is handled in data_hash_keys, so we're probably fine :).